subprojects/core/src/main/java/org/gradle/internal/snapshot/PhysicalSnapshotVisitor.java

```diff
@@ -17,7 +17,7 @@
 package org.gradle.internal.snapshot;
 
 /**
- * Visitor for {@link PhysicalSnapshot}.
+ * Visitor for {@link FileSystemLocationSnapshot}.
```

Comment:
I would call it `FileSystemSnapshotVisitor` since we visit the `FileSystemSnapshot` which consists of `FileSystemLocationSnapshot`s.

Reply:
I guess you meant `FileSystemLocationSnapshotVisitor`? I'd keep `FileSystemSnapshotVisitor`, because it's accepted by a `FileSystemSnapshot`: it lines up very nicely actually.